index.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<p>Simulating <strong>a </strong>grid of stars?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>